portfolio/src/main/webapp/app.js

```diff
@@ -64,10 +64,25 @@ function searchBooks(book, url)
         var publishedDate = info.publishedDate;
         var publisher = info.publisher;
         var maturityRating = info.maturityRating;
+
+        var fieldNameArray = ["title", "genre", "categories", "author", "language", "description", "infoLink", "pageCount", "publishedDate", "publisher", "maturityRating"];
+        var fieldContentArray = [title, genre, categories, author, language, description, infoLink, pageCount, publishedDate, publisher, maturityRating];
+        var fieldsString;
+
+        //passing values through as URL parameters
+        fieldNameArray.forEach(function(field,index){
+           if(index == 0){
+             fieldsString = "?"+field+"="+fieldContentArray[index];
+           }else if(index<fieldNameArray.length-1){
+             fieldsString += "&"+field+"="+fieldContentArray[index]
+           }
+        })
+        fetch("/data"+fieldsString,{method:"POST"});
  
```

Comment:
Adds each field by creating a variable for each and then setting the value of that variable to the one that matches from the longer list. Passing this data as parameters in the URL. 

Later impl idea: move all the variable declarations straight into array.
ex. instead of var maturityRating = info.maturityRating;
      we have [..., info.maturityRating];